uber/automated_emails.py

```diff
@@ -17,14 +17,15 @@ class AutomatedEmail:
         Group: lambda session: session.query(Group).options(subqueryload(Group.attendees))
     }
 
-    def __init__(self, model, subject, template, filter, *, when=(),
+    def __init__(self, model, subject, template, filter, ident, *, when=(),
                  sender=None, extra_data=None, cc=None, bcc=None,
-                 post_con=False, needs_approval=True, ident=None, allow_during_con=False):
+                 post_con=False, needs_approval=True, allow_during_con=False):
 
```

Comment:
Here we are promoting `ident` from an optional keyword parameter to a required positional parameter. This is the change that will require every other plugin that sends automated emails to be updated.